google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/Acl.java

```diff
@@ -146,6 +154,64 @@ static Entity fromPb(Access access) {
     }
   }
 
+  /**
+   * Class for a BigQuery Dataset entity. Objects of this class represent a dataset from a different
```

Comment:
Maybe clarify this is a Dataset ACL entity, not a Dataset entity?

Reply:
Updated. Thanks!